### DISCUSSION TITLE
Issue Assignment

### DISCUSSION THREAD

**Discussion by elyna-grapstein** [earthlab-education/Earth-Analytics-AY24#227] [Q&A] [answered]
Hi all! I joined the course late, and as I'm trying to catch up, I'm realizing that I don't have any issues assigned to me (Scavenger Hunt, reading assignments, etc). How might I go about fixing this?

Thanks, and looking forward to the rest of the course!

**Comment by nquarder** [comment, marked as answer]
Hi @elyna-grapstein and welcome to the course! I will assign issues to you so you can get started. 

Also, have you specified which discussion section you would like to join? If you haven't already, please specify using this Google Form: 
https://docs.google.com/forms/d/e/1FAIpQLScQ54eVXHoudj9bOaM498VkBbNPMPTyJCRDdVi3R0uY9GFpXg/viewform

**Comment by eculler** [comment]
Welcome @elyna-grapstein, I notice you are attending the Thursday 4pm section with @lijo8146. I'm going to close this but feel free to open it back up if you have any further questions!
